src/mainmenu.asm

```diff
@@ -1696,31 +1709,37 @@ ramwatch_lock_right:
 action_ramwatch_edit_left:
 {
     LDA !ram_watch_left : TAX
-    LDA !ram_watch_bank : BEQ .bank7E
+    LDA !ram_watch_write_mode : BEQ +
+    %a8()
++   LDA !ram_watch_bank : BEQ .bank7E
     CMP #$0001 : BEQ .bank7F : BRA .bankSRAM
```

Comment:
I doubt this works in 8-bit mode.  I expect the SNES to interpret CMP #$0001 as CMP #$01 followed by a 00 BRK instruction.

Reply:
Also I've never done this before
Also don't look at this commit nothing to see here, this obviously fixes nothing
https://github.com/tewtal/sm_practice_hack/pull/73/commits/339dfcbe21c2efd7acdbfbbc8cf9c1216b852067

Reply:
Also I could be wrong, programming is fun

Reply:
Interesting. I wonder if there's something that makes it compile differently sometimes. It's working fine on my custom build but crashes in this version. I'll change the `CMP #$0001` to a `DEC` to fix it.